bin/tsv_for_genome_upload.py

```diff
@@ -363,18 +372,27 @@ def process_tax_file(self, filename, type):
                     line = line.strip().split('\t')
                     if type == 'euks':
                         lineage[line[0]] = line[3]
+                        if line[3] == 'Unclassified':
```

Comment:
Does the eukaryotic taxonomy annotator also return "Unclassified"? Or is there another equivalent?
(I thought only GTDB was returning Unclassified...)

Reply:
In other words - should we check for an equivalent in the eukaryotic annotations?

Reply:
good catch, I'm not aware of it. Let's assume they don't have unclassified (until we meet them at some point)